docs/design/Liqo-Agent.md

```diff
@@ -0,0 +1,103 @@
+# Liqo Agent
+## Overview
+**Liqo Agent** is the Liqo component that lets the user interact with Liqo in an easy, friendly way.
+It represents one of the two main entry point to the Liqo environment, along with the **LiqoDash**.
+
+_Agent_ is accessible from its tray icon and provides a menu trough which users can:
+
+* monitor the **status** of their own Liqo
+* get **quick access** to Liqo **main features**, e.g. start/stop, Liqo working mode,
+peering management
+* manage the Agent settings, e.g. notifications
+
+_Agent_ aims at enhancing the Liqo users' experience, especially for non-pro ones, 
+allowing them to perform simple operations without needing to interact with the **LiqoDash**.
```

Comment:
I would replace LiqoDash with Kubectl. Normally, everybody can use a dashboard without training, but will require some exercise to deal with kubectl